idaes/models/properties/general_helmholtz/components/__init__.py

```diff
@@ -26,4 +26,8 @@
     "co2": co2,
     "r134a": r134a,
     "r1234ze": r1234ze,
+    "H2O": h2o,
```

Comment:
Is there a way we could catch this later? E.g. wherever the key is used do `key.lower()` to catch any form of capitalization?

Reply:
I can add a function call as an additional layer.  I guess I was just being lazy here.